Task2/isogram.go

```diff
@@ -0,0 +1,17 @@
+package isogram
+
+import "strings"
+
+func IsIsogram(str string) bool {
+	lowStr := strings.ToLower(str)
+	for i := 0; i < len(lowStr); i++ {
```

Comment:
This looks good!
We could use one loop here, and  we could use implementation of set

in golang we could use map as set, for ex. `var set = map[string]bool{}`
and in first repeating character we return false: 
```
if set[letter] {
	return false
}
```